client/src/api/UserService.ts

```diff
@@ -0,0 +1,20 @@
+import { AxiosResponse } from 'axios'
+import axios from '../core/axios'
+
+export default class UserService {
+    static async getUsers(): Promise<AxiosResponse> {
+        return axios.get('api/user')
+    }
+
+    static async createUser(
+        email: string,
+        password: string
+    ): Promise<AxiosResponse> {
+        const body = {
+            email,
+            password,
+        }
+
+        return axios.post('http://localhost:5000/api/user', body)
```

Comment:
`http://localhost:5000` url shouldn't be written manually

Reply:
corrected